src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyStageFileItem.scala

```diff
@@ -113,7 +115,7 @@ object EasyStageFileItem {
     sdoDir.mkdir()
     for {
       mime <- Try{s.format.get}
-      _ <- writeJsonCfg(sdoDir, JSON.createFileCfg(s.dsLocation.getOrElse(s.unsetUrl), mime, parent, s.subordinate))
+      _ <- writeJsonCfg(sdoDir, JSON.createFileCfg(s.datastreamLocation.getOrElse(s.unsetUrl), mime, parent, s.subordinate))
```

Comment:
Right, if you must. For the `cfg.json` I would still like to stick to the one-to-one mapping to Fedora API parameter names.